lib/content-negotiation.js

```diff
@@ -6,11 +6,13 @@
 'use strict';
 var accepts = require('accepts');
 var debug = require('debug')('strong-error-handler:http-response');
-var sendJson = require('./send-json');
-var sendHtml = require('./send-html');
-var sendXml = require('./send-xml');
```

Comment:
We prefer to keep all `require` calls at the top level. Perhaps rename `sendJson` to `defaultSendJson` (etc.)?

Reply:
Your suggestion would make me happier than this current method.